packages/react-native-aztec/example/package.json

```diff
@@ -1,8 +1,30 @@
 {
   "name": "example",
   "version": "1.0.0",
-  "description": "Example for running ponies",
+  "description": "Example for running ponies.",
```

Comment:
@hypest or @koke - should it stay this way? :)

Reply:
I think it's just a sample app to demo usage of the component, but I don't recall ever using it myself. I think it made more sense when this package was in its own repo, but now it might be better to deprecate it in favor of a story when we get storybooks working on native.

Reply:
Sounds great, let's ignore it in the review process then 👍

Reply:
I'll get rid of it as part of this PR.

Reply:
I got rid of the package.json and yarn.lock file, we'll remove the example folder in another PR